ctapipe/reco/sklearn.py

```diff
@@ -575,19 +934,43 @@ def __call__(self, telescope_type, table):
         ):
             train = table[train_indices]
             test = table[test_indices]
-            cv_prediction, truth, metrics = self.cross_validate(
-                telescope_type, train, test
-            )
-            predictions.append(
-                Table(
-                    {
-                        "cv_fold": np.full(len(truth), fold, dtype=np.uint8),
-                        "tel_type": [str(telescope_type)] * len(truth),
-                        "prediction": cv_prediction,
-                        "truth": truth,
-                    }
+
+            if isinstance(self.model_component, DispReconstructor):
+                (
+                    cv_prediction_norm,
+                    cv_prediction_sign,
+                    truth_norm,
+                    truth_sign,
+                    metrics,
+                ) = self.cross_validate(telescope_type, train, test)
+
+                predictions.append(
+                    Table(
+                        {
+                            "cv_fold": np.full(len(truth_norm), fold, dtype=np.uint8),
+                            "tel_type": [str(telescope_type)] * len(truth_norm),
+                            "norm_prediction": cv_prediction_norm,
```

Comment:
same here: just use one column each for prediction and truth. Using two columns does not provide additional information.